python/ray/data/_internal/stats.py

```diff
@@ -407,6 +418,11 @@ def to_string(
             out += indent
             out += "* Extra metrics: " + str(self.extra_metrics) + "\n"
         out += str(self.iter_stats)
+        if len(self.stages_stats) > 0 and add_global_stats:
+            out += "\nGlobal memory:\n"
+            out += "* Spilled: {}MB\n".format(round(self.global_bytes_spilled / 1e6))
+            out += "* Restored: {}MB\n".format(round(self.global_bytes_restored / 1e6))
```

Comment:
```suggestion
            out += "\nCluster memory:\n"
            out += "* Spilled to disk: {}MB\n".format(round(self.global_bytes_spilled / 1e6))
            out += "* Restored from disk: {}MB\n".format(round(self.global_bytes_restored / 1e6))
```

Can you also update this to only add the string if one of these values is nonzero?